public/app/themes/justice/search.php

```diff
@@ -71,14 +71,14 @@
     $postId = $result->id;
     $filesize = null;
     $format = null;
-    $url = $search->formattedUrl(get_the_permalink($postId));
+    $url = get_the_permalink($postId);
 
     // If the post type is document get the format and filesize
     if ($document->verify_post_type($postId)) {
         $upload_dir = $document->document_upload_dir();
         $year_month = str_replace('-', '/', substr($result->post_date, 0, 7));
         $format = $document->get_file_type($postId);
-        $document_url = "{$upload_dir}/{$year_month}/{$result->post_title}{$format}";
+        $document_url = "{$upload_dir}/{$year_month}/{$result->slug}{$format}";
         if (file_exists($document_url)) {
             $filesize = size_format(wp_filesize($document_url));
```

Comment:
While this code wasn't introduce in this PR, I've spotted something that might be a bug.

As we are offloading documents to S3, they won't be on the filesystem - I think checking the file size like this won't work.

However, WP Offload Media does save the file size in the database, there is a function that you can either use directly or copy. This screenshot should give an overview of the function and database structure.

<img width="1495" alt="image" src="https://github.com/user-attachments/assets/fdc16e33-da40-46db-adb1-bed7a74caee0">